src/config/dynamic-theme-fixes.config

```diff
@@ -2910,6 +2910,15 @@ INVERT
 
 ================================
 
+guitarcenter.pl
+
+INVERT
+tr > td > img
```

Comment:
So inverting the products images?

Reply:
No, they use white images as section separators in the product descriptions. By inverting them they become black instead of white.